pallets/parachain-staking/src/lib.rs

```diff
@@ -507,6 +538,12 @@ pub mod pallet {
 	pub(crate) type CandidateInfo<T: Config> =
 		StorageMap<_, Twox64Concat, T::AccountId, CandidateMetadata<BalanceOf<T>>, OptionQuery>;
 
+	#[pallet::storage]
+	#[pallet::getter(fn candidate_rounds_info)]
+	/// Stores the last round in which a collator produced blocks
+	pub(crate) type CandidateRoundsInfo<T: Config> =
+		StorageMap<_, Twox64Concat, T::AccountId, CandidateLastRound<RoundIndex>, OptionQuery>;
```

Comment:
```suggestion
		StorageMap<_, Blake2_128Concat, T::AccountId, CandidateLastRound<RoundIndex>, OptionQuery>;
```
Twox64Concat should never be used for "user input" keys. It's something that we should change for other storage maps

Reply:
I'd rather be too cautious than not cautious enough, but you're right in this particular context it's probably safe to stay on TwoX :)